src/Control/Parallel.purs

```diff
@@ -48,3 +50,13 @@ parSequence_
   => t (m a)
   -> m Unit
 parSequence_ = parTraverse_ id
+
+parOneOf
+  :: forall a t m f
+   . Parallel f m
+  => Plus f
```

Comment:
For example, `parOneOf [] <*> x = parOneOf []` if you have `Alternative`, but not necessarily otherwise. Also `parOneOf (xs <> ys) <*> x = (parOneOf xs <*> x) <> (parOneOf ys <*> x)` if you have `Alternative`.